src/main/java/org/gridsuite/filter/server/AbstractFilterRepositoryProxy.java

```diff
@@ -43,12 +43,35 @@ static SortedSet<String> setToSorterSet(Set<String> set) {
         return CollectionUtils.isEmpty(set) ? null : new TreeSet<>(set);
     }
 
+    static Map<String, Set<String>> convert(FreePropertiesFilterEntity entity) {
+        if (entity == null) {
+            return null;
+        }
+
+        Map<String, Set<String>> ret = entity.getFreePropertyFilterEntities().stream()
+            .collect(Collectors.toMap(FreePropertyFilterEntity::getPropName, FreePropertyFilterEntity::getPropValues));
+
+        return ret;
```

Comment:
you can do
return entity.getFreePropertyFilterEntities().stream()
            .collect(Collectors.toMap(FreePropertyFilterEntity::getPropName, FreePropertyFilterEntity::getPropValues));

Reply:
I can inline these pre-return variables.
Though I see more inconvenience at debug stage than I can see any real advantage beyond calming linter claims.

Reply:
Done